src/Microsoft.OData.Client/Serialization/HttpWebRequestMessage.cs

```diff
@@ -1,601 +0,0 @@
-﻿//---------------------------------------------------------------------
-// <copyright file="HttpWebRequestMessage.cs" company="Microsoft">
-//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
-// </copyright>
-//---------------------------------------------------------------------
-
-namespace Microsoft.OData.Client
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Diagnostics;
-    using System.Diagnostics.CodeAnalysis;
-    using System.IO;
-    using System.Globalization;
-    using System.Net;
-#if PORTABLELIB
-    using System.Reflection;
-    using System.Xml;
-#endif
-    using Microsoft.OData;
-
-    /// <summary> IODataRequestMessage interface implementation. </summary>
-    [SuppressMessage("Microsoft.Design", "CA1001:TypesThatOwnDisposableFieldsShouldBeDisposable", Justification = "Returning MemoryStream which doesn't require disposal")]
-    public class HttpWebRequestMessage : DataServiceClientRequestMessage

```

Comment:
This seems like a major breaking change that I am not too sure we should have if we are not targeting a major version.

Reply:
fixed